new/LIT5885HymnMary.xml

```diff
@@ -6,7 +6,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
         <fileDesc>
             <titleStmt>
                 <title xml:lang="gez" xml:id="t1">ኦእግዝእትየ፡ ማርያም፡ ጽዋዓ፡ ልቡና፡ መለኮታዊ።</title>
-                <title xml:lang="gez" corresp="#t1" type="normalized">ʾO-ʾǝgzǝʾǝtǝya Māryām ṣǝwāʿā lǝbbunā malakotāwi</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">ʾO-ʾǝgzǝʾǝtǝya Māryām ṣǝwwāʿā lǝbbunā malakotāwi</title>
```

Comment:
Since the lemma is ጽዋዕ (https://betamasaheft.eu/Dillmann/?mode=none&q=%E1%8C%BD%E1%8B%8B%E1%8B%95&id=L87cfd1c1bf4c4396bfe3cd8f4b281407), it seems to me more correct to write ጽዋዐ፡ and ṣǝwwāʿa with final short /a/. Is there a reason to leave ṣǝwwāʿā with final long /a/ in the title?